utils/cmake/dreamcast.cmake

```diff
@@ -92,7 +92,6 @@ function(kos_add_romdisk target romdiskPath)
         OUTPUT  ${obj}
         DEPENDS ${obj_tmp}
         COMMAND ${KOS_CC_BASE}/bin/sh-elf-gcc -o ${obj} -r ${obj_tmp} -L${KOS_BASE}/lib/dreamcast -Wl,--whole-archive -lromdiskbase
-        COMMAND rm ${obj_tmp}
```

Comment:
Without this line, will the romdisk ever get updated if the contents within it change?

Reply:
Well yes. Because `${obj_tmp}` will change.

But that code will be dropped anyway if https://github.com/KallistiOS/KallistiOS/pull/641 is merged...